src/cmd/cmd.cpp

```diff
@@ -478,7 +478,12 @@ int main(int argc, char **argv) {
 
     SyncEngine engine(account, options.source_dir, folder, &db);
     engine.setIgnoreHiddenFiles(options.ignoreHiddenFiles);
+#if QT_VERSION >= QT_VERSION_CHECK(5, 0, 0)
+    QObject::connect(&engine, &SyncEngine::finished,
+                     [&app](bool result) { app.exit(result ? EXIT_SUCCESS : EXIT_FAILURE); });
```

Comment:
Is it sure that we will have c++11 on all our platforms already? Otherwise maybe better guard it with the same defines as in folderstatusmodel.cpp.
`QCoreApplication::exit` is static, so you would probably also not need to capture `app`.